skglm/estimators.py

```diff
@@ -70,7 +69,8 @@ def _glm_fit(X, y, model, datafit, penalty):
         raise ValueError("X and y have inconsistent dimensions (%d != %d)"
                          % (n_samples, y.shape[0]))
 
-    if not model.warm_start or not hasattr(model, "coef_"):
+    # if not model.warm_start or not hasattr(model, "coef_"):
+    if not solver.warm_start or not hasattr(model, "coef_"):
```

Comment:
we probably need an example in the doc of fitting a GLE with warm start, as it will be surprising for sklearn users that warm_start is an attribute of the solver rather than of the model

Reply:
+1 for the warm start example, do you want to do it in this PR or in a separated one?

Reply:
next PR would be OK, this one is big and hanging for a while already